apollo-backend/src/main/java/io/logz/apollo/dao/StackDao.java

```diff
@@ -9,11 +9,9 @@
 import java.util.List;
 
 public interface StackDao {
-    Stack getStack(int id);
+    StackType getStackType(int id);
     EnvironmentsStack getEnvironmentsStack(int id);
     ServicesStack getServicesStack(int id);
-    List<Stack> getAllStacks();
```

Comment:
These are not used?

Reply:
This is not a valid method because Stack is an abstract class, so no, nobody is using it :)